mautic/mautic
# Fix the CKEditor scrolling issue

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes [#10737](https://github.com/mautic/mautic/issues/10737)  



#### Description:
This PR fixes the bug a long email jumps/scrolls down when editing. 

This PR uses the way proposed [here](https://github.com/mautic/mautic/issues/10737#issuecomment-1157418764).

#### Steps to test this PR:
1. Open the email editor of a long email
2. Scroll to the middle
3. Edit a text and press enter

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open the email editor of a long email
3. Scroll to the middle
4. Edit a text and press enter
5. The bug is gone


<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11491"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
